README.md

```diff
@@ -1,6 +1,6 @@
 ![](/docs/source/.static/img/nucypher.png)
 
-*A proxy re-encryption network to empower privacy in decentralized systems*
+*A decentralized cryptological network offering accessible, intuitive, and extensible runtimes and interfaces for secrets management, secure data exchange, and secure computation.*
```

Comment:
```suggestion
*A decentralized cryptological network offering accessible, intuitive, and extensible runtimes and interfaces for secrets management, secure data exchange, dynamic access control, and secure computation.*
```

Reply:
Although perhaps it's better to just replace `secure data exchange` with `dynamic access control`?

Reply:
^ +1 on the replacement.

However, do we want to keep `secure computation` in there - should we remove it? Same comment for other locations.